packages/shared/src/schemas.ts

```diff
@@ -146,10 +151,12 @@ export type FillOrKillParamsWithMinPrice = Omit<FillOrKillParamsX128, 'minPriceX
   minPrice: string;
 };
 
+export type AssetSymbol = (typeof assetConstants)[ChainflipAsset]['rpcAsset'];
```

Comment:
we can also just use `RpcAsset` from `@chainflip/utils` here or is that maybe ambiguous?

Reply:
it's not exported, i can do a new release with the export added

Reply:
Ah true, i just noticed that. Its also incomplete (doesn't have the hub assets), so it would be nice to add them and release anyway. 🙏

Reply:
all the assethub assets are there, `DOT`, `USDC`, and `USDT`

Reply:
indeed they are, excusez-moi